### DISCUSSION TITLE
skynet开源交流社区 qq群 188323917

### DISCUSSION THREAD

**Discussion by G0561** [cloudwu/skynet#1024] [Show and tell]
大家好， 今天发现， 以前有个大群被解散了， 好多同学突然要加入skynet 开源交流社区群，这里说明一下，这个群建立于2014年，欢迎广大新手老手 skyneter 加入讨论技术，  对，是讨论技术！
qq群号： 188323917  需要的朋友来！

**Comment by samoyedsun** [comment]
多谢
